### PR TITLE
60-70% complete

### PR DESCRIPTION
**Pull request by MiTo0o**

### CODE REVIEW

File: command-line-local-shortener/main.go [hunk 1]
```diff
@@ -92,27 +75,11 @@ func dataProccess() {
...
-
-	jsonFile.Close()
 	fmt.Println(id)
-	for i := range dataArray {
-		if dataArray[i].UrlID == id {
-			fmt.Println("yes")
-			c.Redirect(http.StatusMovedPermanently, dataArray[i].LongURL)
-			break
-		}
+	_, exists := dataArray[id]
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Fix this double lookup

File: command-line-local-shortener/main.go [hunk 2]
```diff
@@ -92,27 +75,11 @@ func dataProccess() {
...
+	if exists {
+		c.Redirect(http.StatusMovedPermanently, dataArray[id])
 	}
-
 }
 
 func routerSetup() *gin.Engine {
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Move this to main function

File: command-line-local-shortener/main.go [hunk 3]
```diff
@@ -128,9 +95,24 @@ func routerSetup() *gin.Engine {
 }
 
 func main() {
+
+	jsonFile, err := os.Open("urlmap.json")
+	if err != nil {
+		fmt.Println(err)
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Conver this to logrus just like in [brad-test](https://github.com/codeday-labs/golang-url-shortener/blob/brad-test/brad-test/main.go#L29)

File: command-line-local-shortener/main.go [hunk 4]
```diff
@@ -128,9 +95,24 @@ func routerSetup() *gin.Engine {
...
+	}
+	defer jsonFile.Close()
+	fmt.Println("Successfully Opened url.json")
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	if err := json.Unmarshal(byteValue, &dataArray); err != nil {
+		log.Println(err)
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Logrus again pls

**Review by BradHacker — CHANGES_REQUESTED** [review]
Left some comments. Pls fix prior to merging. Will re-review prior to merge.

File: command-line-local-shortener/main.go (hunk 3 above)
Comment (MiTo0o) [reply command-line-local-shortener/main.go]: oh right

**Review by BradHacker — APPROVED** [review]

**Pull request merged**